Doxygen is still looking for some files:

Doxygen version: 1.14.0
Opening '/Users/runner/work/OpenMS/OpenMS/OpenMS/bld/doc/doxygen/doxygen-error.log' to check for doxygen errors...
----------- ERRORS/WARNINGS -----------------
/Users/runner/work/OpenMS/OpenMS/OpenMS/src/topp/QuantmsIOConverter.cpp:70: warning: included file 'TOPP_QuantmsIOConverter.cli' is not found. Check your EXAMPLE_PATH

I probably missed some part:
https://github.com/search?q=repo%3AOpenMS%2FOpenMS%20QuantMSIOConverter&type=code

--- a/doc/doxygen/public/TOPP.doxygen
+++ b/doc/doxygen/public/TOPP.doxygen
@@ -28,7 +28,9 @@
   - @subpage TOPP_IDFileConverter - Converts between different identification file formats.
   - @subpage TOPP_MSstatsConverter - Convert to MSstats input file format.
   - @subpage TOPP_MzTabExporter - Exports various XML formats to an mzTab file.
+/// @cond WITH_PARQUET
   - @subpage TOPP_QuantmsIOConverter - Converts IdXML files to parquet format following quantms.io PSM specification.
+/// @endcond
   - @subpage TOPP_TargetedFileConverter - Converts targeted files (such as tsv or TraML files).
   - @subpage TOPP_TextExporter - Exports various XML formats to a text file.
   - @subpage TOPP_TriqlerConverter - Convert to Triqler input file format.

--- a/src/topp/QuantmsIOConverter.cpp
+++ b/src/topp/QuantmsIOConverter.cpp
@@ -71,7 +71,7 @@ PEP scores are automatically detected from metavalues using known PEP score name
 @verbinclude TOPP_QuantmsIOConverter.cli
 <B>INI file documentation of this tool:</B>
 @htmlinclude TOPP_QuantmsIOConverter.html
-
+/// @endcond
 */
 
 // We do not want this class to show up in the docu:

--- a/src/topp/QuantmsIOConverter.cpp
+++ b/src/topp/QuantmsIOConverter.cpp
@@ -20,6 +20,7 @@ using namespace std;
 //-------------------------------------------------------------
 
 /**
+/// @cond WITH_PARQUET
 @page TOPP_QuantmsIOConverter QuantmsIOConverter
 
 @brief Converts IdXML files to parquet format following quantms.io PSM specification.

